googlepeople.json

```diff
@@ -1497,36 +1493,24 @@
             },
             {
                 "module": "requests_oauthlib",
-                "input_file": "wheels/shared/requests_oauthlib-1.3.1-py2.py3-none-any.whl"
+                "input_file": "wheels/shared/requests_oauthlib-2.0.0-py2.py3-none-any.whl"
             },
             {
                 "module": "rsa",
                 "input_file": "wheels/py3/rsa-4.9-py3-none-any.whl"
             },
             {
                 "module": "setuptools",
-                "input_file": "wheels/py3/setuptools-69.0.2-py3-none-any.whl"
-            },
-            {
-                "module": "six",
-                "input_file": "wheels/shared/six-1.16.0-py2.py3-none-any.whl"
-            },
-            {
-                "module": "soupsieve",
-                "input_file": "wheels/py3/soupsieve-2.5-py3-none-any.whl"
+                "input_file": "wheels/py3/setuptools-59.6.0-py3-none-any.whl"
```

Comment:
remove this whole pip section? We do not want this version of setuptools that auto-populates here